core-api/src/test/java/com/optimizely/ab/notification/NotificationCenterTest.java

```diff
@@ -70,7 +70,7 @@ public void onActivate(@Nonnull Experiment experiment, @Nonnull String userId, @
     public void testAddActivateNotification() {
         int notificationId = notificationCenter.addActivateNotificationListener(new ActivateNotificationListenerInterface() {
             @Override
-            public void onActivate(@Nonnull Experiment experiment, @Nonnull String userId, @Nonnull Map<String, String> attributes, @Nonnull Variation variation, @Nonnull LogEvent event) {
+            public void onActivate(@Nonnull Experiment experiment, @Nonnull String userId, @Nonnull Map<String, ?> attributes, @Nonnull Variation variation, @Nonnull LogEvent event) {
```

Comment:
Will this break current usage of the notification listeners? If someone has already implemented notification listeners with just `Map<String, String>`, will they suddenly see build errors?

Reply:
It would break current usage (I had to make changes in unit test and the java-testapp). Does it make sense to leave it as Map<String, String> though?

Reply:
Let's hold off on this until we discuss it today. My inclination is that we leave `Map<String, String>`, and in our notification center, coerce all values to String types

Reply:
From today's discussion -- looks like we're using `Map<String, ?>`

Reply:
Yep we are.